backend/services/deployments/tests/common.py

```diff
@@ -376,21 +377,26 @@ def clean_minio(s3_bucket):
     return s3_bucket
 
 
-def mongo_cleanup(mongo):
-    dbs = mongo.list_database_names()
-    dbs = [
-        d
-        for d in dbs
-        if d not in ["local", "admin", "config", "deployment_service", "workflows"]
-    ]
-    for d in dbs:
-        mongo.drop_database(d)
-    db = mongo["deployment_service"]
-    collections = db.list_collection_names()
-    for c in collections:
-        if c == "migration_info":
-            continue
-        db[c].delete_many({})
+def mongo_cleanup(mongo: MongoClient):
+    dbs = mongo.list_databases(
+        filter={"name": {"$nin": ["admin", "config", "local", "workflows"]}},
```

Comment:
for my knowledge (I simply don't remember) - why do we skip `workflows`?

Reply:
I believe it's because of the workflows being persisted in the database. If the collection is dropped it will forget about all the workflows definitions.